packages/apps/graph/src/utils/default.ts

```diff
@@ -0,0 +1 @@
+export const defaultFungibleName: string = 'coin';
```

Comment:
I'd say let's make this an env var. For example when people want to deploy their own graph specific for their own fungible. Otherwise they have to edit and build the code from scratch

Reply:
Makes sense! Will make the change